### ISSUE TITLE
Could not load package matlab-mode

### ISSUE THREAD

**Issue by z24** [jwiegley/use-package#178]
[matlab-mode](http://matlab-emacs.cvs.sourceforge.net/viewvc/matlab-emacs/matlab-emacs/) / 3.3.2

<pre>
(use-package matlab-mode
  :ensure t)
</pre>

**Comment by jwiegley** [comment]
Since I don't use matlab, can you please describe the problem and what you see?

**Comment by thomasf** [comment]
For one thing I do not see a matlab-mode.el in the directory linked in the issue.. 
Try dropping the -mode suffix.

**Comment by z24** [comment]
@thomasf You are right. Thanks! After dropping -mode suffix, it works. I think the reason is that, package.el stores  whole matlab mode files under a directory named `matlab-mode`, but there is no matlab-mode.el file in that directory. Therefore use-package cannot locate correct file to be loaded. But previous version of use-package works well. This issue actually happened once I checked out latest use-package.

@jwiegley Problem has been solved. Thanks!
